### ISSUE TITLE
Feature Request: Button/Slider tab for better touchscreen experience 

### ISSUE THREAD

**Issue by fcebreros** [artisan-roaster-scope/artisan#603]
With touchscreens sometimes buttons and sliders are to small to operate with comfort and if the screen isn't
big enough the graph gets all cluttered up.
A separate tab/window with only buttons and sliders that could be bigger would be a great Hui improvement for users that
want to control and record parameters from inside Artisan.
Thanks
Frank C.

**Comment by sebiik** [comment]
Agreed. Also because space is limited on most touch screen UIs, a tap and hold effect or popup window which magnifies the slider(s) and lets the user operate it would be even fancier.
